NEWS.md

```diff
@@ -2,6 +2,7 @@ htmltools 0.5.1.9000
 --------------------------------------------------------------------------------
 
 * Closed #197: Fixed rendering of boolean attributes in <script> tags rendered via renderDependencies() (#197, thanks @atusy).
+* Closed #204: doRenderTags() now works when provided a top-level tagFunction(). (#204)
```

Comment:
Should be `Closed #203`

Reply:
Thanks, fixed by c200de2d083cfda49949e0ae75abccd5f8d816ff